### PR TITLE
add license files for bower dependencies

### PR DESCRIPTION
**Pull request by mrahanjam**
This PR adds the license files for the bower dependencies in the UI.

### CODE REVIEW

File: cdap-distributions/src/COPYRIGHT/angular-bootstrap@1.1.2/COPYRIGHT [hunk 1]
```diff
@@ -0,0 +1 @@
+The MIT License (MIT)
```
Comment (dli357) [inline cdap-distributions/src/COPYRIGHT/angular-bootstrap@1.1.2/COPYRIGHT:1]: Is this intended to be a single-line file?

File: cdap-distributions/src/COPYRIGHT/ngInfiniteScroll@1.2.1/COPYRIGHT [hunk 2]
```diff
@@ -0,0 +1 @@
+The MIT License (MIT)
```
Comment (dli357) [inline cdap-distributions/src/COPYRIGHT/ngInfiniteScroll@1.2.1/COPYRIGHT]: Is this intended to be a single-line file?

**Review by dli357 — APPROVED** [review]
LGTM, minor comments

File: cdap-distributions/src/COPYRIGHT/ngInfiniteScroll@1.2.1/COPYRIGHT (hunk 2 above)
Comment (mrahanjam) [reply cdap-distributions/src/COPYRIGHT/ngInfiniteScroll@1.2.1/COPYRIGHT]: Addressed

File: cdap-distributions/src/COPYRIGHT/angular-bootstrap@1.1.2/COPYRIGHT (hunk 1 above)
Comment (mrahanjam) [reply cdap-distributions/src/COPYRIGHT/angular-bootstrap@1.1.2/COPYRIGHT:1]: There is no license info on the repo for this dependency, I assumed based on other angular based dependencies that this is also MIT licensed. see [this link](https://github.com/angular-ui/bootstrap-bower)

**Pull request merged**